.github/workflows/issue-opened.yml

```diff
@@ -3,13 +3,30 @@ name: Issue Opened Triage
 on:
   issues:
     types: [opened]
+  workflow_dispatch:
+
+permissions:
+  actions: none
+  checks: none
+  contents: none
+  deployments: none
+  issues: write
+  packages: none
+  pull-requests: none
+  repository-projects: none
+  security-events: none
+  statuses: none
 
 jobs:
   issue_triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: github/issue-labeler@v2
+    - name: Checkout code
+      uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+    - name: Label new issue
+      uses: github/issue-labeler@829a8bf1b184f74cc575b5435093a92c7b846983
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
         configuration-path: .github/labeler-issue-triage.yml
```

Comment:
I'm not sure if the issue labeler will work, since it requires a github token. From what I've observed so far, we don't allow access to any secrets unless you're running from the main branch. The behavior on issues and PRs is unknown. (I don't have access to view or configure those settings, but it seems fine to just see what happens).